provider/cluster_rosa_classic_resource.go

```diff
@@ -698,14 +715,31 @@ func (r *ClusterRosaClassicResource) getAndValidateVersionInChannelGroup(ctx con
 	return "", fmt.Errorf("version %s is not in the list of supported versions: %v", version, versionList)
 }
 
-func (r *ClusterRosaClassicResource) validateAccountRoles(ctx context.Context, state *ClusterRosaClassicState) error {
-	r.logger.Debug(ctx, "Validating if cluster version is compatible to account roles' version")
-	region := state.CloudRegion.Value
-	version, err := r.getAndValidateVersionInChannelGroup(ctx, state)
+func validateHttpTokensVersion(ctx context.Context, logger logging.Logger, state *ClusterRosaClassicState, version string) error {
+	if common.IsStringAttributeEmpty(state.HttpTokensState) {
+		return nil
+	}
+
+	version = getOcmVersionMinor(version)
+	a, err := ver.NewVersion(version)
 	if err != nil {
-		return fmt.Errorf("Could not get cluster version: %v", err)
+		return fmt.Errorf("version '%s' is not supported: %v", version, err)
+	}
+	b, _ := ver.NewVersion(lowestHttpTokensVer)
```

Comment:
we already have a common logic in the function checkMinSupportedVersion
Can you reuse it? or adapting that function to be compatible with your validation?